gazebo/rendering/Visual.cc

```diff
@@ -3635,7 +3648,7 @@ void Visual::AddPendingChild(std::pair<VisualType,
 }
 
 /////////////////////////////////////////////////
-void Visual::ProcessMaterialMsg(const msgs::Material &_msg)
+void Visual::ProcessMaterialMsg(const ignition::msgs::Material &_msg)
 {
   if (_msg.has_lighting())
```

Comment:
This line fails with:

```
/var/lib/jenkins/workspace/gazebo-ci-pr_any-ubuntu_auto-amd64-gpu-nvidia/gazebo/gazebo/rendering/Visual.cc:3653:12: error: ���const class ignition::msgs::Material��� has no member named ���has_lighting���; did you mean ���set_lighting���?
   if (_msg.has_lighting())
            ^~~~~~~~~~~~
            set_lighting
```

This is because proto 3 not having optional fields anymore. 

On `gazebo11`, we should use the `gazebo::msgs` version, which users proto 2. See this commit by @scpeters : https://github.com/osrf/gazebo/commit/7d79b37f790f9b076ff9c73b6561787fd3408b37